packages/server/graphql/private/typeDefs/DeleteFeatureFlagSuccess.graphql

```diff
@@ -0,0 +1,6 @@
+type DeleteFeatureFlagSuccess {
+  """
+  The deleted feature flag
+  """
+  featureFlag: FeatureFlagNew!
```

Comment:
Once I remove the existing `FeatureFlag`, I'll change this from `FeatureFlagNew` to `FeatureFlag`